### DISCUSSION TITLE
Contributor call: 9 July 2022

### DISCUSSION THREAD

**Discussion by joostdecock** [freesewing/freesewing#2300] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:
Location: https://discord.freesewing.org (meeting room: 🔊voice chat)
Date: 9 July 2022 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/jul-09-2022/4pm).
Or check the bot in Discord! #meetings
The meetings will always be at 6pm, Antwerp time.

### Agenda
- Newsletter issues/bugs/problems/frustrations/happy endings
- New in v2.21
- Fantastic development environments and where to find them
- All designs (where relevant) support drafting to high bust now
- Why do we talk so much more than we act? And how can we improve that?
  - Newsletter
  - #1512
  - #2344
  - #2207  
  - #1575 
- The new `nextjs-only` tag, milestones, and feature request freeze
- Is `r/freesewing` maintained? And how about our other communication channels?

> This discussion will be updated after the call with notes and to-dos.

**Comment by bobgeorgethe3rd** [comment]
I'm not sure if this is meeting material but it may be something to discuss. I was thinking for the new site would it be an idea to make the cover arts when being hovered over display the line drawing? I saw this on a few commercial patterns websites and thought that it may be a good UI/UX idea. I know this does mean possible update to the line drawings which I recall an issue about and that is a lot of work if that is needed so this may be a as we update sort of deal but I feel like it would be a really good change from a user end but that may just be me hence meeting topic.

**Reply by joostdecock** [reply]
New line drawings are coming anyway (for the new site).
It's not a bad idea, but we'd have to see how we do it mobile where there is no such thing as *hover*.

**Comment by l3xander** [comment, marked as answer]
### Notes
- @joostdecock explained that version 2.21 was rushed in order to have it finished before the newsletter, but as a result some things are undercooked. We will try to not have this happen again.
- [New stuff in 2.21](https://github.com/freesewing/freesewing/blob/develop/CHANGELOG.md), amongst other things: five new patterns, new development environment and better build speed. ✨
- So, there's a new development environment! It is now the standard as the old one is deprecated, but that one is still available. The new environment makes it easier for makers to connect their designs to FreeSewing.
- Drafting a pattern to the high bust measurement instead of the chest circumference may give more flattering results to those with a bigger bust. Teagan was the first pattern that supported this.
Now @joostdecock has added support for this measurement in all patterns (where it makes sense). 🙌
- @joostdecock raised the issue of, what is holding people back from using Github?
    - There have been Github issues where volunteers have done a lot of work, but don't make a pull request to add it to the site. Copy pasting information is way easier than gathering and writing information, yet people don't do it. Why? Is GitHub scary? Well, to some, yes.
    - @/eriese (from Discord) suggested we motivate people to make a pull request over an issue. @bobgeorgethe3rd added that people should know: if something goes wrong, other people will step in and help you.
- @joostdecock’s new tag for issues: nextjs-only. It indicates we want to implement the feature proposed, but on the new site. On a tagged issue, a robot comes and explains this. 🤖
- @joostdecock noted that the last post on our subreddit is six months old. What do we do with inactive social media? 🤔
    - @joostdecock keeps an eye on the Twitter, what about our other social media? Instagram’s most recent post is from four months ago. We’re not sure about Facebook (”mEtA”). TikTok is non-existent basically.
    - @/eriese from Discord suggested to put something in the description along the lines of, “we’re not really active on this platform, but we’re active here,” with a link to Discord, so people don’t get the wrong idea.
    - Reddit has options for integrations and bots that could automatically make posts. Instagram and Facebook are connected via ‘the metaverse’, which we could make use of.
- @woutervdub is working on octoplushies 👀 It has gone through multiple prototypes already and needs a couple more weeks.
- @/eriese from Discord is working on the cutting lay-out of pattern pieces, a tool for figuring out how much fabric is needed for a pattern, and is wondering, what do folks need from it?
    - @/eriese has been trying to implement that users can input the width and length of the fabric. Similar to the printing lay-out where it will generate more/less pages if you move the pieces around.
    - It takes into account pattern pieces you have to cut on fold, these are mirrored so they’re visible on the fabric. Plus, pieces that have to be cut out multiple times, will appear on the fabric that amount of times. All of this helps to make sure you have the right amount of fabric. 🙌
    - @woutervdub suggested to support grainline, a switch that makes it so you can’t flip a pattern around. Especially helpful for patterned fabrics, this avoids that the pattern would be upside down.
    - @bobgeorgethe3rd followed up on that with, some sewing patterns need directional stretch (e.g. a fabric that stretches more vertically than horizontally). This would work similarly.

### To-do's
- Check up on our social media and decide what we do per platform

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
